packages/lib-classifier/src/store/feedback/helpers/metadata-to-rules.js

```diff
@@ -15,7 +15,8 @@ function metadataToRules (metadata = {}) {
     return result
   }, [])
 
-  return rules.filter(Boolean)
+  const rulesValues = Object.values(rules)
```

Comment:
I don't understand why this works, as I think `rules` is an array, not an object.

Reply:
If this is the rules from the store, it's an observable array... which is an object. 🙃 IF this truly needs to be a vanilla array, then you should convert it first, `Array.from(rules)`

Reply:
Oh this comment was outdated, so doesn't apply anymore, but yeah that's something to remember about data types wrapped by observables.